package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri May 20 08:09:04 UTC 2016 - mchandras@suse.de
+
+- Split SuSEFirewall and SuSEFirewallServices classes to separate files
+  as recommended by the Yast coding style.
```

Comment:
well, if you do not add there FATE number it won't be accepted by IBS as it require for each change have BNC or FATE number.